src/main/java/io/zold/api/Network.java

```diff
@@ -45,4 +50,53 @@
      * @return The wallet
      */
     Wallet pull(Long id);
+
+    /**
+     * Simple network implementation.
+     */
+    final class Simple implements Network {
+
+        /**
+         * {@link Remote} nodes.
+         */
+        private final Iterable<Remote> nodes;
+
+        /**
+         * Constructor.
+         * @param remotes Remotes of the network
+         */
+        Simple(final Iterable<Remote> remotes) {
+            this.nodes =  remotes;
+        }
+
+        @Override
+        public void push(final Wallet wallet) {
+            new HeadOf<Remote>(
+                1,
```

Comment:
@paulodamaso this scoring algorithm is used only when paying taxes. For regular payments, a wallet can be pushed to any arbitrary node

Reply:
@llorllale Right, how do we distinghuish between tax payment and regular payments?

Reply:
@paulodamaso we haven't got to that part yet. For now, I think it would be ok just to push the wallet to all networks. We'll worry about taxes a little later on. You can open a ticket for that actually.

Reply:
@llorllale Right, removed the code in `Network.Simple` but left the tests, they can be used after correct implementation.